felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/Brevmottaker.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.kontrakter.felles.tilbakekreving
+
+import javax.validation.Valid
+import javax.validation.constraints.Pattern
+
+data class Brevmottaker(
+    val type: MottakerType,
+    @field:Pattern(regexp = "^(.{1,80})\$", message = "Feltet kan ikke være tomt eller innholde mer enn 80 tegn")
+    val navn: String,
+    @field:Valid
+    val manuellAdresseInfo: ManuellAdresseInfo
+)
+
+enum class MottakerType(val visningsnavn: String) {
+    BRUKER_MED_UTENLANDSK_ADRESSE("Bruker med utenlandsk adresse"),
+    FULLMEKTIG("Fullmektig"),
```

Comment:
Lurer på om vi også kan droppe å ha fullmektig som en alternativ mottakerType, hvis det er sånn som dette [favro-kortet](https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=NAV-11859) sier, at både verge og fullmektig skal mappes til eksisterende vergetyper i familie-tilbake...
```suggestion
```
I så fall trenger ikke det nye feltet for manelle brevmottakere å være en liste heller. egentlig, mtp. hva som er gyldige kombinasjoner i ba-sak.

Reply:
må nok ha fullmektig her, siden det er snakk om manuelle adresser, fullmektig på verge baserer seg på adresser fra PDL